doc/api/cli.md

```diff
@@ -340,6 +358,16 @@ misformatted, but any errors are otherwise ignored.
 Note that neither the well known nor extra certificates are used when the `ca`
 options property is explicitly specified for a TLS or HTTPS client or server.
 
+### `SSL_CERT_DIR=dir`
+
+If `--use-openssl-ca` is enabled, this overrides and sets OpenSSL's directory
+containing trusted certificates.
+
+### `SSL_CERT_FILE=file`
```

Comment:
One concern that I have with the use of the environment variables for this is that it is easily possible for a rogue module to overwrite the value for child processes. Obviously rogue modules running untrusted code is a completely different problem in and of itself but I'm not convinced that allowing the trust root to be modified in this way is something we should allow. I'm good with the command-line arguments because those are much more explicit.